File: app/src/main/java/com/ruzhan/rxrepository/network/RemoteApi.java
package com.ruzhan.rxrepository.network;

import com.ruzhan.rxrepository.model.UserModel;

import io.reactivex.Single;
import retrofit2.http.GET;

/**
 * Created by ruzhan123 on 2018/6/21.
 */
public interface RemoteApi {

    @GET("ruzhan123/RxRepository/master/data/remote.json")
    Single<UserModel> getRemoteUser();
}

File: app/src/main/java/com/ruzhan/rxrepository/network/RemoteApi.kt
package com.ruzhan.rxrepository.network

import com.ruzhan.rxrepository.model.UserModel

import io.reactivex.Single
import retrofit2.http.GET


interface RemoteApi {

    @get:GET("ruzhan123/android-repository/master/data/remote.json")
    val remoteUser: Single<UserModel>
}
